app/transformation/TransformSigel.java

```diff
@@ -71,19 +93,21 @@ static void process(String startOfUpdates, int intervalSize,
 					splitFileOpener.process(path.toString());
 				});
 		splitFileOpener.closeStream();
-	}
 
-	private static void splitUpSigelDump() {
-		final FileOpener dumpFileOpener = new FileOpener();
-		dumpFileOpener//
-				.setReceiver(new XmlDecoder())//
-				.setReceiver(new XmlElementSplitter(DUMP_TOP_LEVEL_TAG, DUMP_ENTITY))//
-				.setReceiver(
-						xmlFilenameWriter(TransformAll.DATA_OUTPUT_DIR, DUMP_XPATH));
-		dumpFileOpener.process(TransformAll.DATA_INPUT_DIR + "sigel.xml");
-		dumpFileOpener.closeStream();
+ 
 	}
 
+	// private static void splitUpSigelDump() {
+	// 	final FileOpener dumpFileOpener = new FileOpener();
+	// 	dumpFileOpener//
+	// 			.setReceiver(new XmlDecoder())//
+	// 			.setReceiver(new XmlElementSplitter(DUMP_TOP_LEVEL_TAG, DUMP_ENTITY))//
+	// 			.setReceiver(
+	// 					xmlFilenameWriter(TransformAll.DATA_OUTPUT_DIR, DUMP_XPATH));
+	// 	dumpFileOpener.process(TransformAll.DATA_INPUT_DIR + "sigel.xml");
+	// 	dumpFileOpener.closeStream();
+	// }
+
```

Comment:
if this is not necessary we should remove the comments

Reply:
+1